### PR TITLE
Add vertical scroll to customization fields in BO order create page

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Fix todo `max height and verical scroll` in order create page. Adds scrollbar to customizable fields block. Before: Screenshot from 2020-02-27 10-26-02 After: Screenshot from 2020-02-27 11-36-46
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | part of #17282
| How to test?  | Add many customization fields to product (example customizable mug). Go to order create page using new link `admin-dev/sell/orders/orders/new` . find customer and search for product with cuztomized fields (example customizable mug). See that in customizations block there is a scrollbar.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17861)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/_orders_create.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#order-creation-container {
+  .custom-fields-container {
+    max-height: 300px;
+    overflow-y: scroll;
+    padding-bottom: 0.9375rem;
```
Comment (NeOMakinG) [inline admin-dev/themes/new-theme/scss/pages/_orders_create.scss:5]: Simple question, wouldn't be better to us the ps-rail like the left sidebar of BO ? Would looks the same on every computers

**Review by NeOMakinG — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/_orders_create.scss (hunk 1 above)
Comment (zuk3975) [reply admin-dev/themes/new-theme/scss/pages/_orders_create.scss:5]: sorry i dont know about that :disappointed: , haven't done much with front end, so idk whats ps-rail :sweat_smile: . Maybe someone else has answer to that ?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/_orders_create.scss:5]: That's the scroll bar used in the menu sidebar of the BO, I never used it too but it looks not difficult :p

**Review by Progi1984 — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/_orders_create.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/_orders_create.scss:5]: FYI, Worked on the scrollbar of menu, It's perfect scrollbar, that would be awesome to also use it for pages

**Comment by SD1982** [comment]
This PR correctly add the vertical scroll to customization fields in BO order create page (migrated) . Thanks @zuk3975 . @matks this todo probably could be removed from the todo list #17282

**Comment by matks** [comment]
Thank you @zuk3975

**Comment by matks** [comment]
Need rebase 😭

**Comment by Progi1984** [comment]
Hello @zuk3975, thanks for your contribution, would you mind to rebase your pull request before merging ? Thanks

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**